codebuild/cd/cross-unix-build.sh

```diff
@@ -6,10 +6,10 @@ set -ex
 
 cd `dirname $0`/../..
```

Comment:
Actually, I think this script is a dead script that no one uses.

Reply:
let's be bold and delete it